### ISSUE TITLE
kachery daemon 1.0.17 does not start

### ISSUE THREAD

**Issue by khl02007** [kacheryhub/kachery-daemon#1]
@magland I get the following error

```
Error: Cannot find module 'daemonVersion'
Require stack:
- /home/kacheryuser/miniconda3/envs/kachery-env/lib/node_modules/kachery-daemon-node/dist/services/DisplayStateService.js
- /home/kacheryuser/miniconda3/envs/kachery-env/lib/node_modules/kachery-daemon-node/dist/startDaemon.js
- /home/kacheryuser/miniconda3/envs/kachery-env/lib/node_modules/kachery-daemon-node/dist/cli.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:924:15)
    at Function.Module._load (node:internal/modules/cjs/loader:769:27)
    at Module.require (node:internal/modules/cjs/loader:996:19)
    at require (node:internal/modules/cjs/helpers:92:18)
    at Object.<anonymous> (/home/kacheryuser/miniconda3/envs/kachery-env/lib/node_modules/kachery-daemon-node/dist/services/DisplayStateService.js:20:41)
    at Module._compile (node:internal/modules/cjs/loader:1092:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1121:10)
    at Module.load (node:internal/modules/cjs/loader:972:32)
    at Function.Module._load (node:internal/modules/cjs/loader:813:14)
    at Module.require (node:internal/modules/cjs/loader:996:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/kacheryuser/miniconda3/envs/kachery-env/lib/node_modules/kachery-daemon-node/dist/services/DisplayStateService.js',
    '/home/kacheryuser/miniconda3/envs/kachery-env/lib/node_modules/kachery-daemon-node/dist/startDaemon.js',
    '/home/kacheryuser/miniconda3/envs/kachery-env/lib/node_modules/kachery-daemon-node/dist/cli.js'
  ]
}
```

**Comment by magland** [comment]
This should now be fixed on 1.0.18

**Comment by khl02007** [comment]
yes it is
